Backend/Models/CommandMessage.cs

```diff
@@ -17,13 +15,14 @@ internal class CommandMessage : Message
 
         private CommandType ParseCommandType()
         {
+            // TODO: Implement parser and commandtypes
```

Comment:
Remove this comment since it will conflict with Akrums PR

Reply:
Done